### PR TITLE
New Hgg low-mass HLT path added to DQM 

### PR DESCRIPTION
**Pull request by lfinco**
Added a new HLT path (for the Hgg low-mass analysis) to the existing DQM code for diphoton triggers.

### CODE REVIEW

File: HLTriggerOffline/Higgs/python/hltHiggsValidator_cfi.py [hunk 1]
```diff
@@ -201,6 +201,7 @@
             "HLT_Diphoton30PV_18PV_R9Id_AND_IsoCaloId_AND_HE_R9Id_NoPixelVeto_Mass55_v",
             "HLT_Diphoton30EB_18EB_R9Id_OR_IsoCaloId_AND_HE_R9Id_NoPixelVeto_Mass55_v",
             "HLT_Diphoton30EB_18EB_R9Id_OR_IsoCaloId_AND_HE_R9Id_PixelVeto_Mass55_v",
+            "HLT_Diphoton30_18_PVrealAND_R9Id_AND_IsoCaloId_AND_HE_R9Id_PixelVeto_Mass55_v",
```
Comment (mtosi) [inline HLTriggerOffline/Higgs/python/hltHiggsValidator_cfi.py]: I think you need v*

did you test this PR ?
can you point to the plots, please ?

Comment (mtosi) [reply HLTriggerOffline/Higgs/python/hltHiggsValidator_cfi.py]: what about
HLT_Diphoton30_18_PVrealAND_R9Id_AND_IsoCaloId_AND_HE_R9Id_NoPixelVeto_Mass55_v* ?

**Comment by mtosi** [comment]
@cmbuild, please test

**Comment by dmitrijus** [comment]
please test

**Comment by dmitrijus** [comment]
@mtosi Is this ready to be merged? I see some unanswered questions.

**Comment by mtosi** [comment]
@lfinco I think you are adding the other path as well, aren't you ?

**Comment by lfinco** [comment]
Yes, now it should be in.

**Comment by lfinco** [comment]
yes, thanks

**Comment by mtosi** [comment]
@cmsbuild, please test

**Comment by mtosi** [comment]
@dmitrijus can we proceed ?

**Comment by dmitrijus** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**